db-client/package.json

```diff
@@ -8,7 +8,7 @@
   "scripts": {
```

Comment:
Incorrect comment from previous PR - the `"version"` field of `db-client` should be the last published `0.3.1` and not `0.0.0` (so that rush will increment it, and not try to re-use already existing `0.0.1`.

Reply:
Good point.  Changed back and editing the change to be a minor bump (which is pretty major pre-1.0.0).